src/generators/csharp/CSharpRenderer.ts

```diff
@@ -76,11 +76,11 @@ export abstract class CSharpRenderer extends AbstractRenderer<CSharpOptions> {
     case 'string':
       return 'string';
```

Comment:
Why not string? I am dummy with C# so I ask 😅

Reply:
String is a reference, which is by default nullable. There is something called nullable types in C# 8, but don't want to touch that too much, as I have no idea what it is yet 😆 https://stackoverflow.com/questions/32853482/how-can-i-make-my-string-property-nullable